plugins/ingestion-kinesis/licenses/aws-json-protocol-NOTICE.txt

```diff
@@ -0,0 +1,14 @@
+OpenSearch (https://opensearch.org/)
+Copyright OpenSearch Contributors
+
+This product includes software developed by
+Elasticsearch (http://www.elastic.co).
+Copyright 2009-2018 Elasticsearch
+
+This product includes software developed by The Apache Software
+Foundation (http://www.apache.org/).
+
+This product includes software developed by
+Joda.org (http://www.joda.org/).
```

Comment:
I think it's arguably wrong to include this content in the `NOTICE` files. In particular, (most of) the OpenSearch Contributors didn't contribute to the AWS SDK.

I notice it's the same as `repository-s3`, but I'm pretty sure that's wrong too. I think the `NOTICE` files are allowed to be empty.

Reply:
i see. i copied and pasted from `repository-s3`, and i can leave it empty